get-pipsi.py

```diff
@@ -47,17 +52,17 @@ def _cleanup():
         except (OSError, IOError):
             pass
 
-    if Popen(['virtualenv', venv]).wait() != 0:
+    if call(['virtualenv', venv]) != 0:
         _cleanup()
         fail('Could not create virtualenv for pipsi :(')
 
-    if Popen([venv + '/bin/pip', 'install', 'pipsi']).wait() != 0:
+    if call([venv + '/bin/pip', 'install', 'pipsi']) != 0:
```

Comment:
This breaks under Windows. Windows has ``Scripts``, ``Include``, and ``Lib`` as subfolders.